MapboxNavigation/NavigationMapView.swift

```diff
@@ -1455,7 +1487,7 @@ extension NavigationMapView {
             highlightedBuildingColorExpression += "%@)"
             
             // TODO: Verify whether extrusion only for specific building or all buildings is required.
```

Comment:
If we're choosing path that we can highlight either all or specific building with different colors I think `TODO` statement can be removed.

Reply:
Deleted